### PR TITLE
Add README in test/smoke 

### PR DESCRIPTION
**Pull request by aman556**
## What this PR does / why we need it

This PR will add a README file under `test/smoke` which will tell how to run smoke-test for the packages as individual and to run tests for all the packages.
## Details for the Release Notes (PLEASE PROVIDE)

```release-note
NONE
```

## Which issue(s) this PR fixes

Fixes: #

## Special notes for your reviewer

None

### CODE REVIEW

File: test/e2e/smoke-test/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Smoke Test for Packages
+
```
Comment (navidshaikh) [inline test/e2e/smoke-test/README.md:2]: May be add a little description about the scope of the smoke tests ?

File: test/e2e/smoke-test/README.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+## To run test for all the packages at once
```
Comment (navidshaikh) [inline test/e2e/smoke-test/README.md:3]:
```suggestion
## Test all packages
```

File: test/e2e/smoke-test/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+- Move to `test/e2e/smoke-test`.
+- Run `make smoke-test`
```
Comment (navidshaikh) [inline test/e2e/smoke-test/README.md:6]:
```suggestion
`make smoke-test`
```

File: test/e2e/smoke-test/README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+## To Run Smoke Tests for individual Package
```
Comment (navidshaikh) [inline test/e2e/smoke-test/README.md:8]:
```suggestion
## Test individual package
```

File: test/e2e/smoke-test/README.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+These smoke tests are for those packages which do not have e2e-test.
```
Comment (navidshaikh) [inline test/e2e/smoke-test/README.md:10]:
```suggestion
```

File: test/e2e/smoke-test/README.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+1. Cert-manager:
+    To run smoke test for cert-manager.
+    - Move to `test/e2e/smoke-test`.
+    - Run `make cert-manager`
```
Comment (navidshaikh) [inline test/e2e/smoke-test/README.md:15]:
```suggestion
1. Cert-manager:  `make cert-manager`
```

File: test/e2e/smoke-test/README.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+    - Move to `test/e2e/smoke-test`.
+    - Run `make contour`
+3. Fluent-bit:
+    To run smoke test for fluent-bit.
+    - Move to `test/e2e/smoke-test`.
+    - Run `make fluent-bit`
+4. Local-path-storage:
+    To run smoke test for local-path-storage.
+    - Move to `test/e2e/smoke-test`.
+    - Run `make local-path-storage`
```
Comment (navidshaikh) [inline test/e2e/smoke-test/README.md:27]: same suggestion as above

**Review by navidshaikh — CHANGES_REQUESTED** [review]

File: test/e2e/smoke-test/README.md (hunk 1 above)
Comment (aman556) [reply test/e2e/smoke-test/README.md:2]: Thanks @navidshaikh for reviewing, Makes sense updating the file.

File: test/smoke/README.md [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+## Test all packages
+
+- `make smoke-test`
+
+## Test individual package
+
+1. Cert-manager:`make cert-manager`
+2. Contour:`make contour`
+3. Fluent-bit:`make fluent-bit`
+4. Local-path-storage:`make local-path-storage`
```
Comment (navidshaikh) [inline test/smoke/README.md:14]:
```suggestion
1. Cert-manager: `make cert-manager`
2. Contour: `make contour`
3. Fluent-bit: `make fluent-bit`
4. Local-path-storage: `make local-path-storage`
```

File: test/smoke/README.md [hunk 9]
```diff
@@ -0,0 +1,14 @@
+# Smoke Test for Packages
+
+ Smoke tests verifies basic package operations like installing and performing a simple use case for the package.
+
+## Test all packages
+
+- `make smoke-test`
```
Comment (navidshaikh) [inline test/smoke/README.md:7]:
```
make smoke-test
```

**Review by navidshaikh — CHANGES_REQUESTED** [review]
looks better, minor nits else lgtm!

Can you also squash the commits?

**Comment by aman556** [comment]
Thanks @navidshaikh Squashed the commits

**Review by navidshaikh — APPROVED** [review]

**Pull request merged**